index.js

```diff
@@ -7,10 +7,13 @@ module.exports = fp(
       root: path.join(__dirname, 'dist'),
       prefix: '/fastify-overview-ui/'
     })
-    fastify.get('/json-overview-ui', async () => fastify.overview())
+    await fastify.get('/fastify-overview-ui', (req, reply) => {
+      reply.sendFile('index.html')
+    })
+    await fastify.get('/json-overview-ui', async () => fastify.overview())
```

Comment:
This is not necessary if we add the `exposeRoute: true` parameter to the `fastify-overview` config object

Reply:
the reason why I did this is because the client code needs to be built against a specific path and I didn't want this plugin to depend on any particular configuration of fastify-overview, but just require that it is installed